packages/react-admin-core/src/table/TableQuery.sc.ts

```diff
@@ -1,14 +1,27 @@
 import styled from "styled-components";
 
 export const ProgressOverlayContainer = styled.div`
-    position: absolute;
+    position: relative;
+    min-height: 500px;
```

Comment:
@MFSepplive wofür genau brauchst du da die `min-height`?
Des schaut (bei mir zumindest) ziemlich komisch aus, wenn die Tabelle größer is, wie der Inhalt.